src/style.css

```
@@ -0,0 +1,75 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  font-family: Avenir, sans-serif;
+}
+
+.search-bar {
+  position: relative;
+  height: 70px;
+  margin-top: 20px;
+}
+
+.search-bar.small {
+  position: relative;
+  height: 42px;
+  margin-top: 20px;
+}
```

```suggestion
.search-bar--small {
  position: relative;
  height: 42px;
  margin-top: 20px;
}
```
Используй модификатор вместо отдельного класса, чтобы не повышать специфичность селектора